AUTHORS

```diff
@@ -165,6 +165,8 @@ Josh Hursey, Indiana University, Oak Ridge National Laboratory, Los Alamos Natio
   jjhursey@open-mpi.org
 Joshua Gerrard
   enquiries@joshuagerrard.com
+  technopiano@gmail.com
+  joshuagerrard+ompi-commit@protonmail.com
```

Comment:
@OMGtechy Are these the emails that you committed with?  We typically only show the emails that show up in `git log`.